src/gic_calculator/gic_principle_interest.py

```diff
@@ -0,0 +1,28 @@
+def total_gic(principal, interest):
+    """
+    Calculate the total amount for a Guaranteed Investment Certificate (GIC) after n years.
+
+    This function computes the total amount by adding the pre-calculated interest to the principal amount.
+    It assumes that the interest has been calculated separately over a certain number of years.
+
+    Parameters:
+    principal (float): The initial amount of money invested in the GIC.
+    interest (float): The total interest earned over the investment period.
+
+    Returns:
+    float: The total amount after the investment period, which is the sum of the principal and interest.
+
+    Example:
+    For a principal of $1000 and interest of $200, the function will return $1200.
```

Comment:
Look great! But I think the Example part should be the example code instead of the sentence?

Reply:
good idea, thanks. I will redo pull request after edit this.